pkgs/by-name/uu/uutils-findutils/package.nix

```
@@ -0,0 +1,46 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "uutils-findutils";
+  version = "0.7.0";
+
+  src = fetchFromGitHub {
+    owner = "uutils";
+    repo = "findutils";
+    tag = finalAttrs.version;
+    hash = "sha256-EEyrXG9jybtYoBvjiXTCNg6/1WPchEGJcldB6Gqgmdc=";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoVendor {
+    inherit (finalAttrs) src;
+    name = "uutils-findutils-${finalAttrs.version}";
+    hash = "sha256-nZOa7O0S9ykFM9sH6aqlAPtv3hWKF/vAXZYNRnjcOj4=";
+  };
+
+  nativeBuildInputs = [
+    rustPlatform.cargoSetupHook
+  ];
+
+  checkFlags = [
+    # assertion failed: deps.get_output_as_string().contains("./test_data/simple/subdir")
+    "--skip=find::tests::test_find_newer_xy_before_changed_time"
+  ];
+
```

```suggestion

  postInstall = ''
    rm $out/bin/testing-commandline
  '';

```

This binary is only used for testing: https://github.com/uutils/findutils/blob/0.7.0/src/testing/commandline/main.rs#L13